Makefile

```diff
@@ -94,14 +81,22 @@ install: node_modules
 run: welcome githooks install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
-node-version:
-	@$(BIN)/check-node-version
-
 # a helper rule to ensure that a specific module is installed,
 # without relying on a generic `npm install` command
 node_modules/%: | node-version
 	@$(NPM) install $(notdir $@)
 
+# but `npm install semver` can't rely on node-version because node-version
```

Comment:
From https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html:

> Occasionally, however, you have a situation where you want to impose a specific ordering on the rules to be invoked without forcing the target to be updated if one of those rules is executed. In that case, you want to define _order-only_ prerequisites. Order-only prerequisites can be specified by placing a pipe symbol (`|`) in the prerequisites list: any prerequisites to the left of the pipe symbol are normal; any prerequisites to the right are order-only:
>
> `targets : normal-prerequisites | order-only-prerequisites`

I must admit, though, that I didn't change it because these words are not enough for me to understand what the `|` actually does :laughing:

Getting rid of the dependency on `node-version` seems to work fine.  I'll push up a commit to do that.